lib/store.js

```diff
@@ -1,33 +1,70 @@
 const { sleep } = require("./helpers");
+const rl = require("./rl");
 
 class Player {
   constructor(name = null, isHuman = null) {
     this.name = name;
     this.isHuman = isHuman;
   }
```

Comment:
(historical note): in languages like python, it's traditional to write a stub method on the base class to tell people what they need to override. It might look something like:

```
getGridChoice(store) {
  throw new Error("Not Implemented. Please override `getGridChoice()` in your subclass of Player");
}
```

I'm not sure what the js best-practices are here though, because I try to avoid writing code that's complicated enough to need classes in pure js. Once we switch to typescript interfaces, this won't be a thing that we need to think about.